mlflow/models/evaluation/base.py

```diff
@@ -1278,20 +1299,25 @@ def evaluate(
           specified, unless the ``evaluator_config`` option **log_model_explainability** is
           explicitly set to ``True``.
 
-    :param model: A pyfunc model instance, or a URI referring to such a model.
+    :param model: One of the following:
+                  - A pyfunc model instance
+                  - A URI referring to a pyfunc model
+                  - A function
+                  - A string name of a column from ``data`` that contains model outputs
```

Comment:
Design decision: How to specify a static dataset?
Option 1: Use the `model` parameter to specify a string name of a column from ``data`` that contains model outputs.

* 👍  no breaking change
* 👎  It looks weird to specify a column name as `model`.

Option 2: Make the `model` parameter optional (breaking change) and specify the model output column name in the evaluator_config dict.

* 👍  feels natural
* 👎  breaking change

Option 3 (preferred): Ask users to pass in a `model=None` to indicate using a static dataset and specify the model output column name in the evaluator_config dict.
* 👍  no breaking change
* 👍  feels natural